extensions/binding/org.eclipse.smarthome.binding.digitalstrom/src/main/java/org/eclipse/smarthome/binding/digitalstrom/internal/lib/manager/impl/DeviceStatusManagerImpl.java

```diff
@@ -672,7 +668,7 @@ public synchronized void sendSceneComandsToDSS(InternalScene scene, boolean call
                 try {
                     Thread.sleep(sleepTime);
                 } catch (InterruptedException e) {
-                    logger.error("An exception occurred", e);
+                    logger.error("An InterruptedException occurred", e);
```

Comment:
Since this will end up in a user's log file, is this really something that he should report? Or is it a rather rare exception that occurs while testing/debugging? In the later case I would vote to reduce the severity to `debug` here.